patches/src/main/kotlin/app/revanced/patches/all/misc/adb/HideAdbPatch.kt

```diff
@@ -0,0 +1,52 @@
+package app.revanced.patches.all.misc.adb
+
+import app.revanced.patcher.extensions.InstructionExtensions.replaceInstruction
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.patches.all.misc.transformation.transformInstructionsPatch
+import app.revanced.util.getReference
+import com.android.tools.smali.dexlib2.Opcode
+import com.android.tools.smali.dexlib2.iface.instruction.formats.Instruction35c
+import com.android.tools.smali.dexlib2.iface.reference.MethodReference
+
+private const val EXTENSION_CLASS_DESCRIPTOR = "Lapp/revanced/extension/all/spoof/adb/SpoofAdbPatch;"
+private const val SETTINGS_GLOBAL_CLASS_DESCRIPTOR = "Landroid/provider/Settings\$Global;"
+
+@Suppress("unused")
+val hideAdbStatusPatch = bytecodePatch(
+    name = "Hide ADB status",
+    description = "Hide enabled development settings and/or ADB",
```

Comment:
The description needs to be fixed for correct grammar. What is "enabled development settings"?

```suggestion
    description = "Hides enabled development settings and/or ADB.",
```

Reply:
```
/**
 * Whether ADB over USB is enabled.
 */
@Readable
public static final String ADB_ENABLED = "adb_enabled";

/**
 * Whether ADB over Wifi is enabled.
 * @hide
 */
@Readable
public static final String ADB_WIFI_ENABLED = "adb_wifi_enabled";


/**
* Whether user has enabled development settings.
*/
@Readable
public static final String DEVELOPMENT_SETTINGS_ENABLED = "development_settings_enabled";
```